scripts/drone/steps/lib.star

```diff
@@ -1023,3 +1024,21 @@ def validate_scuemata_step():
             './bin/linux-amd64/grafana-cli cue validate-schema --grafana-root .',
         ],
     }
+
+def ensure_cuetsified_step():
+    return {
+        'name': 'ensure-cuetsified',
+        'image': build_image,
+        'depends_on': [
+            'validate-scuemata',
+        ],
+        'commands': [
+            './bin/linux-amd64/grafana-cli cue gen-ts --grafana-root .',
+            '# The above command generates Typescript files (*.gen.ts) from all appropriate .cue files.',
+            '# It is required that the generated Typescript be in sync with the input CUE files.',
+            '# ...Modulo eslint auto-fixes...:',
+            './node_modules/.bin/eslint . --ext .gen.ts --fix',
```

Comment:
Is this an acceptable way to invoke typescript linting? Seems to be what we do with husky in package.json

Reply:
Not sure if prettier (gofmt for Typescript) is run by cuetsy, otherwise "prettier --write" might be a good idea to run to keep all formatting the same?

Reply:
Yeah this is what confused me - @ryantxu mentioned using yarn [here](https://github.com/grafana/grafana/pull/39694#issuecomment-928725349), but we use `eslint` directly in pre-commit, so that strikes me as the truest arbiter of what the expected final committed state.

Reply:
`so that strikes me as the truest arbiter of what the expected final committed state` as English is not my first language, the wordings in this sentence makes me translate the individual words so I can understand the underlying meaning. Not sure I understand tbh, can you use simpler words?

Reply:
so looking at package.json it looks like we only use `prettier` on `{json,scss,md,mdx}` files. that surprises me, i thought we ran it for everything. in that case, i think what you've got is fine for now (it's the same as what we do to all the other typescript files anyway)